weather/jsonld-contexts/weather.jsonld

```diff
@@ -0,0 +1,10 @@
+{
+    "@context": {
+        "observedBy": "https://uri.fiware.org/ns/data-models#observedBy",
+        "cloud": "https://uri.fiware.org/ns/data-models#cloud",
+        "visibility": "https://uri.fiware.org/ns/data-models#visibility",
+        "precipitationIntensity": "https://uri.fiware.org/ns/data-models#precipitationIntensity",
+        "relativeHumidity90": "https://uri.fiware.org/ns/data-models#relativeHumidity90",

```

Comment:
don't forget to update this

Reply:
Yes, wanted to make sure we agree on the model by itself before updating the context file(s).
@AHABID, are you happy with this model as well ?

Reply:
Yes, the model seems to be greate

Reply:
I've updated Contexts.
Can we still use "observedAt" and "observedBy" ?
I noticed in the smartData.Weather context, there's a "dataProvider" and "dateObserved" ...

Reply:
`observedAt` is part of the NGSI-LD specification, so yes we can, and even must.

`dataProvider` could be a replacement for `observedBy`, but as we widely use `observedBy`, I suggest to stick to it.